client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -304,17 +304,14 @@ const MetadataEditor = createClass({
 						onChange={(e)=>this.handleRenderer('V3', e)} />
 					V3
 				</label>
-
-				<a href='/legacy' target='_blank' rel='noopener noreferrer'>
-					Click here to see the demo page for the old Legacy renderer!
-				</a>
+				<small><a href='/legacy' target='_blank' rel='noopener noreferrer'>Click here to see the demo page for the old Legacy renderer!</a></small>
 			</div>
 		</div>;
 	},
 
 	render : function(){
 		return <div className='metadataEditor'>
-			<h1 className='sectionHead'>Brew</h1>
+			<h1 className='sectionHead'>Properties Editor</h1>
```

Comment:
This is the title of the section, not the whole tab (i.e along with "authors" and "privacy")

Reply:
I believe it’s fine the way I have it. Many things in life have an H1, then some content, then are followed with h2 and more content. 

Nothing is really lost by removing “brew” from the top here. Everything in that panel is about the brew. We don’t alter anything else with the units like the account. 

But, we do frequently reference the Properties Editor when helping users, and it’d be nice to have it labeled (beyond the hover state on the bar). 

Just my two cents

Reply:
If you want a top-level title that says "Properties Editor", that is fine. But I meant that having this element on the same level as "Authors" and "Privacy" implies that only part of this panel is the Properties Editor, when "Properties Editor" should apply to the whole page. We are missing one higher-level header where "Properties Editor" belongs.

An equivalent menu at a restaurant:

### Sandwiches (shouldn't this be "Breads", and "Sandwiches" at a higher level at the top?)

- Whole Wheat
- Pumpernickle
- White

### Vegatables

- Lettuce
- Tomato

### Meats

- Turkey
- Bacon

Reply:
Okay, so just leave Properties Editor as H1, and demote "Privacy" etc to H2

Reply:
What about this:

<img width="724" alt="image" src="https://github.com/user-attachments/assets/407ba541-d4fc-4dd8-8ecb-bc196ff210c7">

Reply:
That's fine, yes. Thanks!